bagitobjecttransfer/bagitobjecttransfer/settings/base.py

```diff
@@ -151,7 +151,7 @@
 MEDIA_URL = '/media/'
 STATIC_URL = '/static/'
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
-STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
+STATIC_ROOT = os.path.join(BASE_DIR, '..', 'static')
```

Comment:
Can you undo this change, and make the following change to `docker_prod.py`:

```python
# Overwrite base.py settings for media and static roots

MEDIA_ROOT = "/app/media"
STATIC_ROOT = "/app/static"
```

Reply:
If we're keeping the path of the static directories fixed, would it make sense to use environment variables for this? We could perhaps use the same environment variable within `compose.prod.yml` and `ngix.conf`.

Reply:
> Can you undo this change, and make the following change to `docker_prod.py`:
> 
> ```python
> # Overwrite base.py settings for media and static roots
> 
> MEDIA_ROOT = "/app/media"
> STATIC_ROOT = "/app/static"
> ```

Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/192/commits/1fc4438dd4d6a3f72ee3aa0fc8131f95bf82ff1c and